app/model/scoreboard.js

```diff
@@ -63,6 +65,75 @@ export default class Scoreboard {
         if(!this.isDone()) {
             this._round.nextRound();
         }
+
+        return this;
+    }
+
+    /**
+     * Mark that there were all hits this round. Assumes start with a fresh round
+     * @returns {Scoreboard} this, for chaining
+     */
+    hitAllThisRound() {
+        if(this._round.isFreshRound()) {
+            for(let dart = 0; dart < THROWS_PER_ROUND; dart++) {
+                this.hit();
+            }
+        }
+
+        return this;
+    }
+
+    /**
+     * Mark that there were all misses this round. Assumes start with a fresh round
+     * @returns {Scoreboard} this, for chaining
+     */
+    missAllThisRound() {
+        if(this._round.isFreshRound()) {
+            for(let dart = 0; dart < THROWS_PER_ROUND; dart++) {
+                this.miss();
+            }
+        }
+
+        return this;
+    }
+
+    /**
+     * Mark that there was only one miss this round (with the rest as hits). Assumes start with a fresh round
+     * @param dartMissed The index of the dart that missed
+     * @returns {Scoreboard} this, for chaining
+     */
+    missOneThisRound(dartMissed) {
```

Comment:
Gonna point out that your missOne and hitOne thing mostly only works if you only have 3 throws per round, because then your only options are: miss none (ie hit all), miss one, miss two (ie hit one), or miss all. It would be more obviously extensible to other throw counts if you did it less, well, cutely - eg, "miss none, miss one, miss two, miss n".

Reply:
yeah. this is a good point. but realistically there will only ever be 3 darts per round. I don't see a huge benefit in going overboard to allow DARTS_PER_ROUND to be fully configurable. That said, I'll be a comment in there saying as much